Jellyfin.Api/Controllers/UserViewsController.cs

```diff
@@ -92,14 +91,33 @@ public class UserViewsController : BaseJellyfinApiController
         fields.Add(ItemFields.DisplayPreferencesId);
         dtoOptions.Fields = fields.ToArray();
 
-        var user = _userManager.GetUserById(userId);
+        var user = _userManager.GetUserById(userId.Value);
 
         var dtos = folders.Select(i => _dtoService.GetBaseItemDto(i, dtoOptions, user))
             .ToArray();
 
         return new QueryResult<BaseItemDto>(dtos);
     }
 
+    /// <summary>
+    /// Get user views.
+    /// </summary>
+    /// <param name="userId">User id.</param>
+    /// <param name="includeExternalContent">Whether or not to include external views such as channels or live tv.</param>
+    /// <param name="presetViews">Preset views.</param>
+    /// <param name="includeHidden">Whether or not to include hidden content.</param>
+    /// <response code="200">User views returned.</response>
+    /// <returns>An <see cref="OkResult"/> containing the user views.</returns>
+    [HttpGet("Users/{userId}/Views")]
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    [Obsolete("Kept for backwards compatibility")]
+    [ApiExplorerSettings(IgnoreApi = true)]
+    public QueryResult<BaseItemDto> GetUserViews2(
+        [FromRoute, Required] Guid userId,
+        [FromQuery] bool? includeExternalContent,
+        [FromQuery, ModelBinder(typeof(CommaDelimitedArrayModelBinder))] CollectionType?[] presetViews,
+        [FromQuery] bool includeHidden = false) => GetUserViews(userId, includeExternalContent, presetViews, includeHidden);
```

Comment:
```suggestion
        [FromQuery] bool includeHidden = false)
        => GetUserViews(userId, includeExternalContent, presetViews, includeHidden);
```

The only suggestion I have is to place the expression body on the next line- when I first read through this I thought it was magic

Reply:
The server is full of crazy linters that complain about all my formatting, so I auto-formatted using Rider.

Reply:
Like really, I don't want to start another tabs vs spaces like war but this is the only sane way to format it and somehow we decided to not do this.

![91700002d0](https://github.com/jellyfin/jellyfin/assets/2305178/c5e68790-a262-4de6-ac86-8ba009471372)

Reply:
don't bring your kotlin into this :1st_place_medal: